resources/testharness.js

```diff
@@ -4710,6 +4710,7 @@
         }
         target[components[components.length - 1]] = object;
     }
+    expose(expose, 'expose');
```

Comment:
Enables use in other helpers to "expose" testing functions in appropriate global scopes.